docs/content/custom-resources/_index.md

```diff
@@ -502,6 +500,13 @@ kubectl apply -f "${pgo_cluster_name}-${pgo_cluster_replica_suffix}-pgreplica.ya
 
 Add this time, removing a replica must be handled through the [`pgo` client]({{< relref "/pgo-client/common-tasks.md#high-availability-scaling-up-down">}}).
 
+### Monitoring
+
+To enable the [monitoring]({{< relref "/architecture/monitoring.md">}})
+(aka metrics) sidecar using the `crunchy-postgres-exporter` container, you need
```

Comment:
Notice we call it the "metrics sidecar" in other places within these changes.

Reply:
Well, we have language for both in this :wink:

Reply:
Oops, definitely missed the `aka` part somehow.  Thanks!